test/service-start.bats

```diff
@@ -73,3 +73,10 @@
   echo "$output"
   [[ $output = *NotFound* ]]
 }
+
+@test "gRPC: attributes example" {
+  run go run ./examples --creds opentdf:secret attributes
+  echo "$output"
+  [ $output = *listing namespaces* ]
+  [ $status = 0 ]
+}
```

Comment:
nit dont have to address: maybe put it in a new bat file, since this one just seems to be testing whether the services are up, and the other is round trip specific, maybe a new one for attribute/policy creation, or it could go in the other one and we just rename the bat idc